pkg/apis/maistra/conversion/conversion_test.go

```diff
@@ -23,6 +22,92 @@ var (
 	cpuUtilization80 = int32(80)
 	intStrInt1       = intstr.FromInt(1)
 	intStr25Percent  = intstr.FromString("25%")
+
+	globalMultiClusterDefaults = map[string]interface{}{
+		"enabled": false,
+		"multiClusterOverrides": map[string]interface{}{
+			"expansionEnabled":    nil,
+			"multiClusterEnabled": nil,
+		},
+	}
+	globalMeshExpansionDefaults = map[string]interface{}{
+		"enabled": false,
+		"useILB":  false,
+	}
+
+	roundTripTestCases = []struct {
+		name   string
+		smcpv1 v1.ControlPlaneSpec
+		cruft  *v1.HelmValues
+	}{
+		{
+			name: "simple",
+			smcpv1: v1.ControlPlaneSpec{
+				Version: "v1.1",
+				Istio: v1.NewHelmValues(map[string]interface{}{
+					"global": map[string]interface{}{
+						"proxy": map[string]interface{}{
+							"image": "asd",
+						},
+						"some-unmapped-field": map[string]interface{}{
+							"foo":   "bar",
+							"fooey": true,
+						},
+					},
+					"prometheus": map[string]interface{}{
+						"enabled": true,
+					},
+					"tracing": map[string]interface{}{
+						"enabled": true,
+					},
+				}),
+			},
+			cruft: v1.NewHelmValues(map[string]interface{}{
+				"global": map[string]interface{}{
+					// a result of enabling tracing
+					"enableTracing": true,
+					// mesh expansion is disabled by default
+					"meshExpansion": map[string]interface{}{
+						"enabled": false,
+						"useILB":  false,
+					},
+					// multicluster is disabled by default
+					"multiCluster": map[string]interface{}{
+						"enabled": false,
+						"multiClusterOverrides": map[string]interface{}{
+							"expansionEnabled":    nil,
+							"multiClusterEnabled": nil,
+						},
+					},
+					// a result of enabling tracing, default provider is jaeger
+					"proxy": map[string]interface{}{
+						"tracer": string("jaeger"),
```

Comment:
this should now be 'zipkin', right?

Reply:
/test unittests

probably, after i rebase

Reply:
That is only used to remove those keys after conversion, so the checks pass, i.e. this documents what keys are added, but not values.